Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -925,7 +925,7 @@ theorem lt_iff' : μ < ν ↔ μ ≤ ν ∧ ∃ s, μ s < ν s :=
   lt_iff_le_not_le.trans <| and_congr Iff.rfl <| by simp only [le_iff', not_forall, not_le]
 
 instance covariantAddLE [MeasurableSpace α] :
-    CovariantClass (Measure α) (Measure α) (· + ·) (· ≤ ·) :=
+    AddLeftMono (Measure α) :=
```

Comment:
This should be merged with the line above, and the instance name should be changed to `instAddLeftMono`.